src/core/qgismobileapp.cpp

```diff
@@ -143,13 +145,19 @@ void QgisMobileapp::initDeclarative()
   qRegisterMetaType<QgsRelation>( "QgsRelation" );
 
 
+  // Register QgsQuick QML types
+  qmlRegisterType<QgsQuickMapCanvasMap>( "org.qgis", 1, 0, "MapCanvasMap" );
+  qmlRegisterType<QgsQuickMapSettings>( "org.qgis", 1, 0, "MapSettings" );
+  qmlRegisterType<QgsQuickCoordinateTransformer>( "org.qfield", 1, 0, "CoordinateTransformer" );
+  qmlRegisterSingletonType<QgsQuickUtils>( "Utils", 1, 0, "Utils", utilsSingletonProvider );
```

Comment:
registering singleton to access static methods
see https://evileg.com/en/post/263/